### ISSUE TITLE
add ability to set label color and label size in the OpenLayerView for the PointMarker

### ISSUE THREAD

**Issue by drewbotts** [opensensorhub/osh-js#564]

**Comment by mdhsl** [comment]
Could you provide more precision about your issue?
Label of marker I guess?
size of? icon? marker? polygon? linestring?

**Comment by drewbotts** [comment]
Specifically labelColor and labelSize in the PointMarker layer. They don't to be used in an OpenLayerView.

**Comment by mdhsl** [comment]
Note: After investigation OL does not support to specify the size of the icon.The property size of the icon does not allow to change the size of the icon itself but The property size of the icon does not allow to change the size of the icon itself but the sub-rectangle to be used from the original icon image (sprite) as specified here (https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html).

OL uses the default icon size; the only way to change the size is to use the scale property which is the ratio of the iconSize property / original size.

The problem is that we know the original size of the icon only after adding it to the DOM, so we see the icon appear at its original size before the scale. So this is not appropriate.

To sum up, OL does not support specifying the size of the icon. You have to use the iconScale property of the PointMarkerLayer and this will work

**Comment by mdhsl** [comment]
I will now work on labelSize & labelColor

**Comment by mdhsl** [comment]
The same comment can be applied to labelSize. I have added a new property to PointMarkerLayer called 'labelScale'. Here the result for the OLVIew:

![image](https://user-images.githubusercontent.com/6577027/151534940-5018129a-1bad-4b70-8bfd-bca7677d1340.png)

**Comment by mdhsl** [comment]
done
